install arduino-cli via homebrew (#437)
arduino/arduino-cli: README.md

## How to install

> ### macOS/Linux installation via HomeBrew

Reviewer: ```suggestion
### macOS/Linux installation via Homebrew
```

Author: done, thanks @dawidd6

> Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is now availabe as a brew formula:

Reviewer 2: ```suggestion
Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is availabe as a brew formula since version `0.5.0`:
```
I would remove the temporal reference, as will be outdated soon and may be confusing, and use a version reference.

Reviewer: Also: `available`. Typo.

> Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is availabe as a brew formula since version `0.5.0`:

Reviewer 3: Hi @Efrat19,
Thanks a lot for your contribution, we appreciate a lot a PR that enhances the ease of installation for the cli!

May I ask you to modify the line below according to the suggestion?

```suggestion
The arduino-cli is available as a brew formula since version `0.5.0`:
```

The `README` is still the main source of documentation for the project and we'd like to keep install instructions simple and direct. We still want to give our contributors more visibility, https://github.com/arduino/arduino-cli/pull/442  is a first step in that direction but we plan to do more and more as the project grows.

Thanks a lot for your contribution!
